podman-rpm-info-vars.sh

```diff
@@ -5,5 +5,5 @@
 # 2. Both PODMAN_VERSION and PODMAN_PR_NUM will have to be updated manually on release
 # PRs.
 # 3. If PODMAN_PR_NUM is empty, rpms will be fetched from the `rhcontainerbot/podman-next` copr.
-export PODMAN_VERSION="5.5.0-dev"
-export PODMAN_PR_NUM=""
+export PODMAN_VERSION="5.4.1"
+export PODMAN_PR_NUM="25535"
```

Comment:
It should not be merged with that on main, that should only be used for testing

Reply:
sure thing